package.json

```diff
@@ -70,50 +70,51 @@
     "vrtest": "cd apps && cd vr-tests && yarn screener"
   },
   "devDependencies": {
-    "@types/jest": "24.9.1",
-    "@types/jest-axe": "3.2.2",
-    "@types/prettier": "1.19.1",
-    "@types/webpack-dev-middleware": "4.1.0",
     "@storybook/addon-a11y": "6.0.28",
-    "@storybook/addons": "6.0.28",
     "@storybook/addon-actions": "6.0.28",
     "@storybook/addon-info": "6.0.0-alpha.2",
     "@storybook/addon-knobs": "6.0.28",
+    "@storybook/addons": "6.0.28",
+    "@storybook/channels": "6.0.28",
     "@storybook/core": "6.0.28",
     "@storybook/react": "6.0.28",
-    "@storybook/channels": "6.0.28",
+    "@types/jest": "24.9.1",
+    "@types/jest-axe": "3.2.2",
+    "@types/prettier": "1.19.1",
+    "@types/storybook__addon-info": "5.2.3",
```

Comment:
I added only this package, all deps were re-ordered by yarn automatically

Reply:
I wouldn't be too picky on what "baggage" comes with 3rd party packages when its not shipped to consumers in general, but in this case I agree - it's deprecated anyways. thx